lms/templates/edxnotes/edxnotes.html

```diff
@@ -102,17 +102,14 @@ <h2 class="placeholder-title">${_('You have not made any notes in this course ye
 </script>
 % endfor
 </%block>
-<%block name="js_extra">
+
 % if notes:
-<script type="text/javascript">
-  (function (require) {
-      require(['js/edxnotes/views/page_factory'], function (NotesPageFactory) {
-          var pageView = new NotesPageFactory({
-              notesList: ${notes},
-              debugMode: ${debug}
-          });
-    });
-  }).call(this, require || RequireJS.require);
-</script>
+    <%block name="js_extra">
+    <%static:require_module module_name="js/edxnotes/views/page_factory" class_name="NotesPageFactory">
+        NotesPageFactory({
+            notesList: ${notes},
```

Comment:
If the user has no notes defined, I get a JS error-- Uncaught ReferenceError: None is not defined.

NotesPageFactory({
            notesList: None,
            debugMode: true
        });